bindings/node/native/src/pre_tokenizers.rs

```diff
@@ -7,6 +7,34 @@ use std::sync::Arc;
 use serde::{ser::SerializeStruct, Serialize, Serializer};
 use tk::pre_tokenizers::PreTokenizerWrapper;
 use tk::PreTokenizedString;
+use tk::normalizer::SplitDelimiterBehavior;
+
+#[derive(Clone)]
+struct JsSplitDelimiterBehavior(SplitDelimiterBehavior);
```

Comment:
I thought the best way is to add a `JsSplitDelimiterBehavior` struct to be able to convert `string` to `SplitDelimiterBehavior`.

The rust code compiles without error, but not very sure if everything is correct here